modules/react/skeleton/lib/parts/skeletonShape.tsx

```diff
@@ -1,23 +1,9 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
-import canvas from '@workday/canvas-kit-react/tokens';
 
-export const BOTTOM_MARGIN = 16;
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import canvas from '@workday/canvas-kit-react/tokens';
 
-const Shape = styled('div')<{
-  width?: number | string;
-  height?: number | string;
-  borderRadius?: number | string;
-  backgroundColor?: string;
-}>(({width, height, borderRadius, backgroundColor}) => {
-  return {
-    width: width ? width : '100%',
-    height: height ? height : '100%',
-    borderRadius: borderRadius ? borderRadius : 0,
-    backgroundColor,
-    marginBottom: BOTTOM_MARGIN,
-  };
-});
+const BOTTOM_MARGIN = 16;
 
 export interface SkeletonShapeProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Same here. Don't need to extend from `React.HTMLAttributes<HTMLDivElement>` anymore.

Reply:
Addressed.